config/environments/production.rb

```diff
@@ -17,42 +14,35 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = false
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
-  config.serve_static_assets = true
-
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger = ActiveSupport::Logger.new(STDOUT)
-    logger.formatter = config.log_formatter
-    config.logger = ActiveSupport::TaggedLogging.new(logger)
-  end
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
-  config.assets.css_compressor = :sass
+  config.assets.js_compressor = :uglifier
+  # config.assets.css_compressor = :sass
```

Comment:
These two changes need to be reverted I think. We are using `sass`, and previously we've needed `harmony: true` to make ES6 syntax work.

Reply:
Is the `harmony` thing an error that shows up during heroku build time? If so could verify it on staging.

Regarding not specifying `:sass`:
<img width="671" alt="Screenshot 2019-11-13 at 09 40 02" src="https://user-images.githubusercontent.com/3461/68746776-97ccd300-05f9-11ea-8ff2-f1a6b0a4254f.png">

This is the default config generated for rails so seems sensible to stick to it unless necessary to deviate.

Reply:
Not sure if it was during build, but sure, give it a try on Staging.